graylog2-web-interface/src/components/lookup-tables/CachesOverview.tsx

```diff
@@ -0,0 +1,145 @@
+import type { PaginationQueryParameterResult } from 'hooks/usePaginationQueryParameter';
+
+import Styles from './Overview.css';
+
+const buildHelpPopover = () => {
+  return (
+    <Popover id="search-query-help" className={Styles.popoverWide} title="Search Syntax Help">
+      <p><strong>Available search fields</strong></p>
+      <Table condensed>
+        <thead>
+          <tr>
+            <th>Field</th>
+            <th>Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>id</td>
+            <td>Cache ID</td>
+          </tr>
+          <tr>
+            <td>title</td>
+            <td>The title of the cache</td>
+          </tr>
+          <tr>
+            <td>name</td>
+            <td>The reference name of the cache</td>
+          </tr>
+          <tr>
+            <td>description</td>
+            <td>The description of cache</td>
+          </tr>
+        </tbody>
+      </Table>
+      <p><strong>Examples</strong></p>
+      <p>
+        Find caches by parts of their names:<br />
+        <kbd>name:guava</kbd><br />
+        <kbd>name:gua</kbd>
+      </p>
+      <p>
+        Searching without a field name matches against the <code>title</code> field:<br />
+        <kbd>guava</kbd> <br />is the same as<br />
+        <kbd>title:guava</kbd>
+      </p>
+    </Popover>
+  );
+};
+
+type Props = {
+  caches: LookupTableCache[],
+  pagination: PaginationType,
+  paginationQueryParameter: PaginationQueryParameterResult,
+};
+
+const CachesOverview = ({ caches, pagination, paginationQueryParameter }: Props) => {
+  const { currentPage, currentPageSize, resetPage } = React.useMemo(() => ({
+    currentPage: paginationQueryParameter.page || 1,
+    currentPageSize: paginationQueryParameter.pageSize || 10,
+    resetPage: paginationQueryParameter.resetPage,
+  }), [paginationQueryParameter]);
+
+  const onPageChange = (newPage: number, newPerPage: number) => {
+    LookupTableCachesActions.searchPaginated(newPage, newPerPage, pagination.query);
+  };
+
+  const onSearch = (query: string, resetLoadingStateCb: () => void) => {
+    resetPage();
+    LookupTableCachesActions.searchPaginated(1, currentPageSize, query).then(resetLoadingStateCb);
+  };
+
+  const onReset = () => {
+    resetPage();
+    LookupTableCachesActions.searchPaginated(currentPage, currentPageSize);
+  };
+
+  return (
+    <Row className="content">
+      <Col md={12}>
+        <h2 style={{ marginBottom: 16 }}>
+          Configured lookup Caches <small>{pagination.total} total</small>
+        </h2>
+        <PaginatedList activePage={currentPage}
+                       pageSize={currentPageSize}
+                       onChange={onPageChange}
+                       totalItems={pagination.total}>
+          <SearchForm onSearch={onSearch} onReset={onReset}>
+            <OverlayTrigger trigger="click" rootClose placement="right" overlay={buildHelpPopover()}>
```

Comment:
Looks like `buildHelpPopover` does not have to be a function call and it can just be the element.

Reply:
Similar case in to `DataAdaptersOverview` and `LookupTablesOverview`.

Reply:
So there is a weird issue with Overlay component, If we pass a component to the overlay prop, the overlay isn't properly positioned.

As a component:

![image](https://user-images.githubusercontent.com/105670363/207894327-49b6968d-6018-4e3b-a45e-48eea29e9b96.png)

As render function

![image](https://user-images.githubusercontent.com/105670363/207894547-e08ef120-4efc-44b7-9f63-a244dce5276e.png)

Reply:
I see, alright!